app/lib/features/profile/pages/my_profile_page.dart

```diff
@@ -176,100 +174,88 @@ class MyProfile extends ConsumerWidget {
         body: SingleChildScrollView(
           child: Column(
             children: [
-              Stack(
-                children: <Widget>[
-                  Wrap(
-                    children: [
-                      Container(
-                        margin: const EdgeInsets.fromLTRB(20, 15, 20, 0),
+              Container(
+                margin: const EdgeInsets.fromLTRB(20, 15, 20, 0),
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(25),
+                ),
```

Comment:
I wonder if we should outsource this repeating boxdecoration into the theme maybe?

Reply:
Like have it in the App Theme ?

Reply:
yeah, like that. probably.